components/widgets/inputs/input/input.tsx

```diff
@@ -98,9 +100,11 @@ function Input({
                     Input_fieldset___legend: showLegend,
                 })}
             >
-                <legend className={classNames('Input_legend', {Input_legend___focus: showLegend})}>
-                    {showLegend ? label || placeholder : null}
-                </legend>
+                {!noLegend && (
```

Comment:
nit: better call it addLegend and prevent the negation condition

Reply:
Perhaps `useLegend`.